.gitignore

```
@@ -1 +1,2 @@
 /*.o
+hello
```

This line will match a file named `hello` in any subdirectory, not just in this directory. Can I suggest this instead?

```suggestion
/hello
```